nupm/utils/dirs.nu

```diff
@@ -37,3 +40,35 @@ export def nupm-home-prompt [] {
 
     mkdir $env.NUPM_HOME
 }
+
+export def script-dir [--ensure]: nothing -> path {
+    let d = $env.NUPM_HOME | path join scripts
+
+    if $ensure {
+        mkdir $d
+    }
+
+    $d
+}
+
+export def module-dir [--ensure]: nothing -> path {
+    let d = $env.NUPM_HOME | path join modules
+
+    if $ensure {
+        mkdir $d
+    }
+
+    $d
+}
+
+export def tmp-dir [subdir: string, --ensure] {
```

Comment:
for consistency with the two other ones :wink: 
```suggestion
export def tmp-dir [subdir: string, --ensure]: nothing -> path {
```

Reply:
done in https://github.com/nushell/nupm/pull/18/commits/49c4cf5fc11aeeb028b14c5a2701e90213b8fbfa